ark/utils/tiff_utils.py

```diff
@@ -0,0 +1,144 @@
+from fractions import Fraction
+import numpy as np
+from skimage.external.tifffile import TiffFile, TiffWriter
+import json
+import datetime
+
+
+def read_mibitiff(file, channels=None):
```

Comment:
We don't need to exhaustively test this, but I think it would be good to have a simple test function that creates a mibi-tiff, reads it in, makes sure the channels are in the right order, etc.